### DISCUSSION TITLE
Does the lighthouse provide URL or entire site?

### DISCUSSION THREAD

**Discussion by brendonguedes** [GoogleChrome/lighthouse#14992] [Q&A] [answered]
Is the Lighthouse assessment done by url+slug or is it a result of an overall assessment of my entire site?

**Comment by adamraine** [comment, marked as answer]
The default Lighthouse mode will only measure a single page navigation on the URL you provide. If you want to test other parts of your site, you can design a [user flow](https://github.com/GoogleChrome/lighthouse/blob/main/docs/user-flows.md).

**Reply by brendonguedes** [reply]
Thanks, Adam!
